### ISSUE TITLE
Bug after upgrading from 0.8.2 to 0.9.0

### ISSUE THREAD

**Issue by o-alexandrov** [JoviDeCroock/prefresh#144]
Webpack doesn't bundle after upgrading:
- `@prefresh/webpack` from `0.8.2` to `0.9.0`

CLI error:
```sh
building ./node_modules/html-webpack-plugin/lib/child-compiler.js:175
    return childCompilation.assets[helperFileName].source();
                                                   ^

TypeError: Cannot read property 'source' of undefined
    at ./node_modules/html-webpack-plugin/lib/child-compiler.js:175:52
    at Array.map (<anonymous>)
    at extractHelperFilesFromCompilation (./node_modules/html-webpack-plugin/lib/child-compiler.js:174:43)
    at ./node_modules/html-webpack-plugin/lib/child-compiler.js:113:13
    at ./node_modules/webpack/lib/Compiler.js:343:11
    at ./node_modules/webpack/lib/Compiler.js:681:15
    at AsyncSeriesHook.eval [as callAsync] (eval at create (./node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:24:1)
    at AsyncSeriesHook.lazyCompileHook (./node_modules/tapable/lib/Hook.js:154:20)
    at ./node_modules/webpack/lib/Compiler.js:678:31
    at AsyncSeriesHook.eval [as callAsync] (eval at create (./node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (./node_modules/tapable/lib/Hook.js:154:20)
    at ./node_modules/webpack/lib/Compilation.js:1423:35
    at AsyncSeriesHook.eval [as callAsync] (eval at create (./node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (./node_modules/tapable/lib/Hook.js:154:20)
    at ./node_modules/webpack/lib/Compilation.js:1414:32
    at AsyncSeriesHook.eval [as callAsync] (eval at create (./node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
```

Yarn's lock changes (no other deps were updated):
<img width="782" alt="Screen Shot 2020-08-09 at 9 37 46 PM" src="https://user-images.githubusercontent.com/9992724/89748393-98ab6c00-da88-11ea-9b03-9f66565d0ca2.png">

**Comment by JoviDeCroock** [comment]
Hey @o-alexandrov 

Thanks for bringing this to my attention, I was passing along something the wrong way, this has been fixed in 0.9.1

**Comment by o-alexandrov** [comment]
@JoviDeCroock Thank you, `0.9.1` doesn't have this issue when bundling w/ `webpack`.
